core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -515,14 +524,20 @@ private void deleteAndVerifyConfig(Admin client, Optional<String> brokerId, Set<
     }
 
     private void verifyConfigDefaultValue(Admin client, Optional<String> brokerId, Set<String> config) throws Exception {
```

Comment:
it is called "verifyConfigDefaultValue" but it takes "broker id". It means the returned configs could be per-broker rather than cluster-wide, right?

Reply:
Yes, It's right this method verify pre-broker level configs

Reply:
> Yes, It's right, this method verify pre-broker level configs

maybe we should rename it?

Reply:
True, I think `verifyPerBrokerConfigValue` is better